pod-rotation-controller/scripts/pod-rotation-controller.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
if you'd ask me, all the scripts in the world should replace this to the following
`#!/usr/bin/env bash` :new_moon_with_face:

Reply:
Good point @simisimis . Updated